packages/hash/api/src/index.ts

```diff
@@ -40,10 +41,26 @@ import { logger } from "./logger";
 import { getRequiredEnv } from "./util";
 import { setupStorageProviders } from "./storage/storage-provider-lookup";
 import { getAwsRegion } from "./lib/aws-config";
+import { setupTelemtry } from "./telemetry/snowplow-setup";
 
 const shutdown = new GracefulShutdown(logger, "SIGINT", "SIGTERM");
 
 const main = async () => {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  let tracker: Tracker | undefined;
```

Comment:
Currently the snowplow `tracker` isn't used in the rest of the code for analytics, ideally we'd add it to the GQL context so we can pass it down wherever we need it.